### ISSUE TITLE
Boost System Library

### ISSUE THREAD

**Issue by DalInar** [CQMP/Maxent#33]
While building Maxent, failed to find the boost system library:

```
Linking CXX executable maxent
CMakeFiles/maxent.dir/src/maxent.cpp.o: In function `__sti___ZSt8__ioinit':
maxent.cpp:(.text+0x3620): undefined reference to `boost::system::generic_category()'
maxent.cpp:(.text+0x362c): undefined reference to `boost::system::generic_category()'
maxent.cpp:(.text+0x3638): undefined reference to `boost::system::system_category()'
make[2]: *** [maxent] Error 1
```

I think the boost::system lib used to be found by ALPSCore and included in the ALPSCore_LIBRARIES, but isn't any more.  Just manually including the system lib in the CMakeLists.txt fixes the issue:

```
target_link_libraries(libmaxent
        /u/shared/ALPSCore_stuff/software/install/lib/libboost_system.so ${ALPSCore_LIBRARIES} ...)
```

But obviously that's not a real solution.  Probably ALPSCore should be modified to find the boost system library like it used to?

**Comment by ryanlevy** [comment]
Are you on the master branch or GPLv3? I think this might be related to #32

**Comment by DalInar** [comment]
Yup, I'm on the Master branch.  Is the GPLv3 branch the one that will develop alongside ALPSCore?

**Comment by galexv** [comment]
> I think the boost::system lib used to be found by ALPSCore and included in the ALPSCore_LIBRARIES, but isn't any more.

It's because ALPSCore does not need them anymore, after commit https://github.com/ALPSCore/ALPSCore/commit/e3849cea4695cee3d76338e14b09f8fcb87058bb .

> Probably ALPSCore should be modified to find the boost system library like it used to?

I think it would be better to modify `CMakeLists.txt` of Maxent to find the modules of Boost it requires -- but it really does not need Boost::FS.

I have just created PR #34 to remove the reference to Boost::FS. Joe (@Dallnar), try if it works for you?

**Comment by galexv** [comment]
Seems to be addressed by PR #34.
